lldb/packages/Python/lldbsuite/test/commands/help/TestHelp.py

```diff
@@ -89,7 +89,7 @@ def test_help_version(self):
         version_str = self.version_number_string()
         match = re.match('[0-9]+', version_str)
         search_regexp = ['lldb( version|-' + (version_str if match else '[0-9]+') + '| \(swift-.*\)).*\n']
-        search_regexp[0] += 'Swift'
+        search_regexp[0] += '[Apple]* Swift'
```

Comment:
This works, but I'm not sure it's really what you want. I think this should be `'(Apple|) Swift'`

Reply:
I think you're right.